akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSMActor.scala

```diff
@@ -17,9 +17,12 @@ import scala.reflect.ClassTag
  * FSM actor implementation with persistent state
  *
  * Supports the usual [[akka.actor.FSM]] functionality with additional persistence features.
- * State and State Data are persisted on every state change.
- * FSM is identified by 'persistenceId' value.
+ * State changes are persisted. FSM is identified by 'persistenceId' value.
```

Comment:
Ok, let's do:

- Remove the section https://github.com/akka/akka/pull/17974/files#diff-ef6f00459ea008459f3774f7875a3e5eR23
- Make the above: "State changes are persisted atomically together with domain events, which means that either both succeed or both fail, i.e. a state transition event will not be stored if persistence of an event related to that change fails".

Is that right?